app/src/main/java/com/example/concordia_campus_guide/view_models/RoutesActivityViewModel.java

```diff
@@ -151,10 +151,11 @@ public void setTransportType(String transportType) {
         this.transportType = transportType;
     }
 
-    public void noShuttles(String noShuttleText) {
+    public void noRoutesOptions(String noRoutes) {
         RouteBuilder routeBuilder = new RouteBuilder();
-        director.constructShuttleRouteWithSummaryOnly(routeBuilder, noShuttleText);
-        Route shuttleRoute = routeBuilder.getRoute();
-        routeOptions.add(shuttleRoute);
+        routeBuilder.setMainTransportType(transportType);
```

Comment:
Can you use the director instead of using the builder directly. That's the proper chain of commands for the builder design pattern.

Reply:
@imabhijit can you take a look at this ?

Reply:
I'm not sure if that is necessary, this is pretty much the only place where we would need to make a Route without anything besides the summary. This route is not "shuttle-only" anymore, it is for any transportation type.

Reply:
anyways, I changed it. Let's follow the code standard!